src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -136,10 +134,12 @@ public CacheHandle lookup(Spawn spawn, SpawnExecutionContext context)
     } catch (CacheNotFoundException e) {
       // There's a cache miss. Fall back to local execution.
     } catch (IOException e) {
-      // There's an IO error. Fall back to local execution.
-      reportOnce(
-          Event.warn(
-              "Some artifacts failed to be downloaded from the remote cache: " + e.getMessage()));
+      String errorMsg = e.getMessage();
+      if (errorMsg == null || errorMsg.length() == 0) {
```

Comment:
errorMsg.isEmpty() instead of length == 0

Reply:
done